.github/workflows/codespell.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [devel]
```

Comment:
Let's run on all branches.

Reply:
```suggestion
```

Reply:
done locally

Reply:
I can't commit this because I would still need to do a rebase to include the change into the commit that introduced `.github/workflows/codespell.yml`

Reply:
no need - already done. I just thought first to do while on the phone, hence did suggestions, but then realized that need to get to laptop anyways.